.env.example

```diff
@@ -1,7 +1,12 @@
-DATABASE_URL=
+DB_USERNAME_DEV=

```

Comment:
We don't need lines 1 - 4. Please remove them

Reply:
oo, okay.
I thought that would remain only on my local but forgot that we synced our .env.example with .env.
It's done @iverenshaguy